indico/modules/events/layout/controllers/menu.py

```diff
@@ -147,16 +172,25 @@ def _process(self):
                     raise BadRequest(f'New parent entry not found for Menu entry "{self.entry}".')
 
             self.entry.insert(parent_entry, position)
-
         else:
             self.entry.move(position)
 
+        log_msg = 'Separator Entry' if self.entry.is_separator else \
+            f'{self.entry.type.title} Entry "{self.entry.localized_title}"'
```

Comment:
Pretty sure `localized_title` returns a translated string, which is not great in log messages (that are untranslated on purpose)